src/3p-frame.js

```diff
@@ -80,11 +80,13 @@ function getFrameAttributes(parentWindow, element, opt_type, opt_context) {
     mode: getModeObject(),
     canary: !!(parentWindow.AMP_CONFIG && parentWindow.AMP_CONFIG.canary),
     hidden: !viewer.isVisible(),
-    amp3pSentinel: generateSentinel(parentWindow),
+    sentinel: generateSentinel(parentWindow),
     initialIntersection: element.getIntersectionChangeEntry(),
     domFingerprint: domFingerprint(element),
     startTime,
   };
+  attributes.ampcontextVersion = (getMode().localDev ? 'LOCAL' :
```

Comment:
I'll move this as part of the #6506 as I am already making the name building helper function there